### PR TITLE
Fixed alignment of bootstrap alert box and added tab icon.

### PR DESCRIPTION
**Pull request by rohitsh16**

### CODE REVIEW

File: templates/profile.html [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
+<div class="sticky-top">
     {% if messages %}
     <ul style="list-style:none;" class="messages">
         {% for message in messages %}
-        <li {% if message.tags %} class="alert alert-danger" {% endif %}>{{ message }}</li>
+        <li {% if message.tags %}class="alert alert-danger text-center" style="width: 95%;" {% endif %}>{{ message }}</li>
         {% endfor %}
     </ul>
-    {% endif %}
+    {% endif %}</div>
```
Comment (JayjeetAtGithub) [inline templates/profile.html]: Indent the code properly here

**Comment by rohitsh16** [comment]
I'll update it.

**Comment by JayjeetAtGithub** [comment]
@rohitsh16 updates on this pr ?

File: templates/profile.html (hunk 1 above)
Comment (rohitsh16) [reply templates/profile.html]: done

**Pull request merged**